lib/secure_sockets/portable/freertos_plus_tcp/aws_secure_sockets.c

```diff
@@ -531,52 +532,56 @@ BaseType_t SOCKETS_Init( void )
  * and FreeRTOS+TCP threads. Assume that two or more threads may race to be the
  * first to initialize the static and handle that case accordingly.
  */
-static CK_RV prvSocketsGetCryptoSession( CK_SESSION_HANDLE * pxSession,
+static CK_RV prvSocketsGetCryptoSession( SemaphoreHandle_t *pxSessionLock,
```

Comment:
NP: this file looks like it needs to be uncrustified.

Reply:
Fixed.